src/overcooked_demo/server/static/js/psiturk.js

```diff
@@ -0,0 +1,222 @@
+// Persistent network connection that will be used to transmit real-time data
```

Comment:
Same with this?

Reply:
This one is technically not in use, though based on the code this setup supports human vs. human game mode, though the setup is more complicated. This is required if the game is setup on the server, though it is not required for AI vs. AI games or AI vs Human games.

Reply:
I guess we'll have a better idea later of how this file interfaces with the files in the other repos!